eos/b-decays/b-to-dstar-l-nu_TEST.cc

```diff
@@ -0,0 +1,156 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2018 Ahmet Kokulu
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <test/test.hh>
+#include <eos/observable.hh>
+#include <eos/b-decays/b-to-dstar-l-nu.hh>
+#include <eos/utils/complex.hh>
+#include <eos/utils/wilson-polynomial.hh>
+
+#include <array>
+#include <cmath>
+#include <iostream>
+#include <fstream>
+#include <limits>
+#include <string>
+#include <vector>
+
+
+using namespace test;
+using namespace eos;
+
+class BToDstarLeptonNeutrinoTest :
+    public TestCase
+{
+    public:
+        BToDstarLeptonNeutrinoTest() :
+            TestCase("b_to_dstar_l_nu_test")
+        {
+        }
+
+        virtual void run() const
+        {
+
+            // tests for normalized SM Br (|V_cb|=1)
+            {
+                Parameters p1 = Parameters::Defaults();
+                /*
+                 * for the TEST case below the B->D^* SSE parameters are randomly chosen. However, the correlations together with the EOM conditions among the FFs are respected in this choice.
+                 */
+                p1["B->D^*::alpha^A0_0@BSZ2015" ] = 1.0;
+                p1["B->D^*::alpha^A0_1@BSZ2015" ] = 0.0;
+                p1["B->D^*::alpha^A0_2@BSZ2015" ] = 0.0;
+                p1["B->D^*::alpha^A1_0@BSZ2015" ] = 0.0;
+                p1["B->D^*::alpha^A1_1@BSZ2015" ] = 0.0;
+                p1["B->D^*::alpha^A1_2@BSZ2015" ] = 0.0;
+                p1["B->D^*::alpha^A12_0@BSZ2015"] = 0.28064568285509967;
+                p1["B->D^*::alpha^A12_1@BSZ2015"] = 0.0;
+                p1["B->D^*::alpha^A12_2@BSZ2015"] = 0.0;
+                p1["B->D^*::alpha^V_0@BSZ2015"  ] = 0.0;
+                p1["B->D^*::alpha^V_1@BSZ2015"  ] = 0.0;
+                p1["B->D^*::alpha^V_2@BSZ2015"  ] = 0.0;
+                p1["B->D^*::alpha^T1_0@BSZ2015" ] = 0.0;
+                p1["B->D^*::alpha^T1_1@BSZ2015" ] = 0.0;
+                p1["B->D^*::alpha^T1_2@BSZ2015" ] = 0.0;
+                p1["B->D^*::alpha^T2_0@BSZ2015" ] = 0.0;
+                p1["B->D^*::alpha^T2_1@BSZ2015" ] = 0.0;
+                p1["B->D^*::alpha^T2_2@BSZ2015" ] = 0.0;
+                p1["B->D^*::alpha^T23_0@BSZ2015"] = 0.0;
+                p1["B->D^*::alpha^T23_1@BSZ2015"] = 0.0;
+                p1["B->D^*::alpha^T23_2@BSZ2015"] = 0.0;
```

Comment:
I think setting (almost) all FF parameters to zero does not sufficiently test the observable(s). Can you
- use a more 'randomly' chosen set of parameters?
- also use this for the BSM test below?